src/react/stylesheets/icons.scss

```diff
@@ -15,15 +15,15 @@
 }
 
 .cc-react-icon--green {
-  color: $cc-color-salem;
+  color: $cc-color-salem !important;
```

Comment:
@developish - This is necessary to override other things that interfere with react component styling?

Reply:
Yes, without this, the icon colors show up as a light gray in the menubar popover.